fmt/bytes.ts

```diff
@@ -17,7 +17,7 @@ type LocaleOptions = {
 };
 
 /**
- * The options for pretty printing the byte numbers.
+ * @depreacted (will be removed after 0.170.0) use `FormatOptions` instead
```

Comment:
This is spelled incorrectly in a few places:

```suggestion
 * @deprecated (will be removed after 0.170.0) use `FormatOptions` instead
```